webpack/resolve-chunk-plugin/src/ResolveChunkPlugin.ts

```diff
@@ -0,0 +1,70 @@
+/// <reference path="./webpackTypes.d.ts" />
+
+import * as Webpack from 'webpack';
+
+// tslint:disable-next-line:variable-name
+const ConstDependency: IConstDependency = require('webpack/lib/dependencies/ConstDependency');
+
+export class ResolveChunkPlugin implements Webpack.Plugin {
+  public apply(compiler: Webpack.Compiler): void {
+    compiler.plugin('compilation', (compilation, data) => {
+      const chunkIdMap: Map<string, ((id: number | undefined) => void)[]> =
+        new Map<string, ((id: number | undefined) => void)[]>();
+
+      data.normalModuleFactory.plugin('parser', (parser) => {
```

Comment:
>parser [](start = 49, length = 6)

typedef?

Reply:
Inferred.

---
In reply to: [139566072](https://github.com/Microsoft/web-build-tools/pull/375#discussion_r139566072) [](ancestors = 139566072)